plugins/by-name/colorizer/default.nix

```
@@ -0,0 +1,251 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "colorizer";
+  originalName = "nvim-colorizer.lua";
+  package = "nvim-colorizer-lua";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  # TODO introduced 2024-07-06: remove after 25.05
+  imports =
+    let
+      oldPluginPath = [
+        "plugins"
+        "nvim-colorizer"
+      ];
+      newPluginPath = [
+        "plugins"
+        "colorizer"
+      ];
+
+      optionsToRename = [
+        "RGB"
+        "RRGGBB"
+        "names"
+        "RRGGBBAA"
+        "AARRGGBB"
+        "rgb_fn"
+        "hsl_fn"
+        "tailwind"
+        "sass"
+        "virtualtext"
+        "virtualtext_inline"
+        "virtualtext_mode"
+        "always_update"
+      ];
+
+      renameOption =
+        optionName:
+        lib.mkRenamedOptionModule
+          (
+            oldPluginPath
+            ++ [
+              "userDefaultOptions"
+              optionName
+            ]
+          )
+          (
+            newPluginPath
+            ++ [
+              "settings"
+              "user_default_options"
+              optionName
+            ]
+          );
+    in
+    (map renameOption optionsToRename)
```

Nit: no need for parens here
```suggestion
    map renameOption optionsToRename
```